internal/plugins/ansible/v1/scaffolds/init.go

```diff
@@ -35,7 +35,7 @@ import (
 
 const (
 	// kustomizeVersion is the sigs.k8s.io/kustomize version to be used in the project
-	kustomizeVersion = "v3.8.7"
+	kustomizeVersion = "v4.5.2"
```

Comment:
who should know what the kustomize version used/support is the plugin responsible to generate/scaffolding the kustomize files since its manifests need to be compatible with. Also, it does not seem to fit under the domain of resposanbility of ansible/helm/go plugins. 

Who is responsible for the kustomize is the kustomize plugin. 
See that we are proposing to fix it in the PR, see:

- https://github.com/kubernetes-sigs/kubebuilder/pull/2583/files#r839541438
- https://github.com/kubernetes-sigs/kubebuilder/pull/2583/files#diff-d0732e49fcd2f9db0172457152d553dac06a3024719b704a613575b6d8982b20R41-R43
- https://github.com/kubernetes-sigs/kubebuilder/pull/2583/files#r839543969

So that, we could do dp the same fix here.

Reply:
You can change your scaffold as please you.
However, to support these archs we need to more than only be able to have kustomize to them, such as:
- All images scaffolds by default must support them
- For golang kubebuilder-tools which ships what is required to use env test also must support

(ihmo) we need to address all changes in Kubebuilder. 
When kubebuilder be able to support then we can update SDK.
We might able to make ansible/helm support first but that would also probably make it hard for us to keep the things maintained.